src/app/(marketing)/profile/_components/OperatedGuilds.tsx

```diff
@@ -85,10 +89,17 @@ export const OperatedGuild = ({ guildBase }: { guildBase: GuildBase }) => {
           <EmphasizedData>{rewardCount}</EmphasizedData>
           rewards in total
         </OperatedGuildDetail>
-        {createdAt && (
+        {createdAt && guild.createdAt && (
```

Comment:
Why do we need both conditions here? 👀 `createdAt` already checks is `guild.createdAt` is defined.